src/sass/_instance_forms.scss

```diff
@@ -36,6 +41,16 @@
       margin-bottom: 6px;
     }
 
+    .profile-action-btn {
+      margin-right: 1rem;
+      margin-top: 2.5rem;
+    }
+
+    .profile-delete-btn {
+      margin-left: 1rem;
```

Comment:
Shouldn't we use vanilla spacing variables when possible? The two horizontal margins here should have `$sph--large` instead of `1rem`.